crates/oxc_linter/src/rules/jsx_a11y/aria_proptypes.rs

```diff
@@ -375,6 +383,7 @@ fn test() {
         r#"<div aria-label="" />"#,
         "<div aria-label />",
         "<div aria-label={`Close`} />",
+        "<div aria-label={`Hello ${foo}`} />",
```

Comment:
I would recommend adding a few extra tests for some minor variations:
```suggestion
        "<div aria-label={`Hello ${foo}`} />",
        "<div aria-label={`foo-${id}`} />",
        "<div aria-label={`foo-${id} bar`} />",
```